src/CattyTests/DataModel/UserVariableTests.swift

```diff
@@ -70,4 +73,91 @@ final class UserVariableTests: XCTestCase {
         XCTAssertFalse(userVariableA === userVariableCopy)
         XCTAssertTrue(userVariableB === userVariableCopy)
     }
+
+    func testIsEqualToUserVariableForDirectAssignment() {
+        let userVariableA = UserVariable()
+        userVariableA.name = "userVar"
+        userVariableA.value = self.arrayWithSameElement
```

Comment:
I think my change request was a bit misleading. I did not mean same elements in one array, but more the following scenario which is important for the method under test. Pseudo code:
```
userVariableA.value = [1, 2]
userVariableB.value = [1, 3]
XCTAssertFalse(userVariableA.isEqual(to: userVariableB))

userVariableB.value = [1, 2]
XCTAssertTrue(userVariableA.isEqual(to: userVariableB))

```